docs/source/deployment/deploy_sre.md

```diff
@@ -69,11 +69,56 @@ sre:
 
 ::::
 
+:::{hint}
+See [here](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/) for a full list of valid Azure VM SKUs
+:::
+
 :::{important}
 All VM SKUs you want to deploy must support premium SSDs.
 
+- SKUs that support premium SSD have a lower case 's' in their name.
+- See [here](https://learn.microsoft.com/en-us/azure/virtual-machines/vm-naming-conventions) for a full naming convention explanation.
 - See [here](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssds) for more details on premium SSD support.
-- See [here](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/) for a full list of valid SKUs
+
+:::
+
+:::{important}
+All VM SKUs you want to deploy must have CPUs with the `x86_64` architecture.
+
+- SKUs with a lower case 'p' in their name have the ARM architecture and should not be used.
+- See [here](https://learn.microsoft.com/en-us/azure/virtual-machines/vm-naming-conventions) for a full naming convention explanation.
+
+:::
+
+:::{important}
+The antivirus process running on each workspace consumes around 1.3 GiB at idle.
+This usage will roughly double for a short period each day while its database is updated.
+
+You should take this into account when choosing a VM size and pick a SKU with enough memory overhead for your workload and the antivirus service.
+:::
+
+:::{important}
+Only GPUs supported by CUDA and the Nvidia GPU drivers can be used.
+['N' series](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/overview#gpu-accelerated) SKUs feature GPUs.
+The NC and ND families a recommended as they feature GPUs designed for general purpose computation rather than graphics processing.
```

Comment:
```suggestion
The NC and ND families are recommended as they feature GPUs designed for general purpose computation rather than graphics processing.
```